ui/app/styles/helper-classes/layout.scss

```
@@ -43,6 +43,10 @@
   position: relative;
 }
 
+.is-absolute {
```

The updated pattern is to avoid prepending with `is-` or `has-`
```suggestion
.position-absolute {
```